src/pages/Intents/DialogCustomIntents.js

```diff
@@ -63,7 +64,7 @@ const DialogCustomIntents = ({
             arguments:
                 specificSettings?.request?.arguments?.length > 0
                     ? specificSettings.request.arguments
-                    : [{ key: '', value: '' }],
+                    : [{ key: '', value: '', id: generateDhis2Id() }],
```

Comment:
@Chisomchima: I added this id because the first was missing an id, but I'm not sure if we want to use id here in the first place? It's not in the specs, so we could also just make the updates based on index of the {key,value} item in the array and skip adding an id in the first place?